cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1439,26 +1419,35 @@ public void testReconcileAllNamespaces(Params params, VertxTestContext context)
 
         // providing the list of ALL StatefulSets for all the Kafka clusters
         Labels newLabels = Labels.forStrimziKind(Kafka.RESOURCE_KIND);
-        when(mockKsOps.list(eq("*"), eq(newLabels))).thenReturn(
-                asList(KafkaCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, bar, VERSIONS).generateStatefulSet(openShift, null, null))
+        when(mockKsOps.list(eq(kafkaNamespace), eq(newLabels))).thenReturn(
```

Comment:
This has to use the `*` in all the code below since that is what means _reconcile all namespaces_.

Reply:
Resolving because ultimately realised I'd switched test names or bodies... ...somehow. I'm assuming while rebasing. Should look a lot better now.